corehq/form_processor/models.py

```diff
@@ -88,4 +124,15 @@ def read_content(self):
             content = f.read()
         return content
 
+
+class XFormOperationSQL(models.Model):
```

Comment:
Also not sure this should be a separate table of just a field in xform. Check out this interesting post: http://schinckel.net/2014/09/24/using-postgres-composite-types-in-django/